lms/djangoapps/discussion/rest_api/serializers.py

```diff
@@ -325,6 +353,7 @@ class CommentSerializer(_ContentSerializer):
     endorsed_at = serializers.SerializerMethodField()
     child_count = serializers.IntegerField(read_only=True)
     children = serializers.SerializerMethodField(required=False)
+    abuse_flagged_any_user = serializers.SerializerMethodField(required=False)
```

Comment:
nit: flagged_abuse is sufficient I think.

Reply:
There is already a field called `abuse_flagged` that is true if the **current** user has flagged the comment, so calling this one `flagged_abuse` will be confusing.